src/common/task_runner.cpp

```diff
@@ -125,7 +125,9 @@ void TaskRunner::Process(const MainloopContext &aMainloop)
     // Critical error happens, simply die.
     VerifyOrDie(errno == EAGAIN || errno == EWOULDBLOCK, strerror(errno));
 
+exit:
     PopTasks();
+    return;
```

Comment:
nit: Per syntax, this 'return' is unnecessary. I'm not sure if having a 'return' here is suggested by the style? @jwhui

Reply:
Removed it. As I remember, we cannot put `exit:` at the end of a function (limited by compiler or pretty tool or code style). But now that we have a statement after `exit:`, we don't need the `return`.